rental.rb

```diff
@@ -4,11 +4,7 @@ class Rental
 
   def initialize(date, person, book)
```

Comment:
- [ ] I can create a rental successfully but when I try to see a list of rentals for a particular person I see none, even after I create a rental with that student. Please check into this.